### DISCUSSION TITLE
My Wi-Fi toggle stop working

### DISCUSSION THREAD

**Discussion by rromney1** [eduardofilo/RG350_adam_image#115] [Hardware problems?]
I have the original RG350. I love the Adam image it works way better than the original stock image but I can't get the Wi-Fi dongle to work on it. The Adam image it will would work for one sec and stop but when I put the original SD with the stock image it works fine is any suggestions?

**Comment by eduardofilo** [comment]
There are differences in the support of different Wifi dongles between stock and the ODbeta used in Adam.

In stock the supported ones were:

* Realtek RTL8188EU 150Mb/s (stock and ROGUE)
* Realtek RTL8188FU 150Mb/s (only ROGUE)
* Realtek RTL8192CU 300Mb/s (stock and ROGUE)

And in ODbeta:

* Realtek RTL8192CU
* Mediatek MT7601U

So, surely your dongle is based in RTL8188. The main developer of ODbeta commented that the RTL8188 chip is not supported in the latest versions of the Linux kernel, so he cannot integrate its support. You will have to find a supported dongle. I'm sorry.

**Reply by SFrost007** [reply]
Is this a change that happened specifically in newer versions of the ODbeta kernel? I have a RTL8188EU adapter for my 350M; this was definitely supported in earlier versions of ODBeta and not just stock/rogue. It works fine on Adam 1.1 (kernel 5.14.0, from Aug 25th) but doesn't on Adam 1.3.1 (kernel 5.15.0, from Oct 15th).

Do you have a link to the comment from the ODBeta developer, so I could maybe figure out the last version of ODBeta which supported this chipset?

**Reply by eduardofilo** [reply]
https://discordapp.com/channels/529983248114122762/871075357422415952/914894799453175808

**Reply by trekker99** [reply]
actually, from this post https://www.phoronix.com/scan.php?page=news_item&px=Linux-5.15-Staging it actually sounds like the driver was changed, not that support was dropped.
